documentation/dev_running_from_source.markdown

```diff
@@ -214,3 +233,50 @@ Debugging
 
 For more information about debugging both Clojure and JRuby code, please see
 [Puppet Server: Debugging](./dev_debugging.markdown) documentation.
+
+Running PuppetDB
+-----
+
+To run a source PuppetDB with Puppet Server, Puppet Server needs
+standard PuppetDB configuration and how to find the PuppetDB
+terminus. First copy the `dev/puppetserver.conf` file to another
+directory. In your copy of the config, append a new entry to the
+`ruby-load-path` list: `<PDB source path>/puppet/lib`. This tells
+PuppetServer to load the PuppetDB terminus from the specified
+directory.
+
+From here, the instructions are similar to installing PuppetDB
+manually via packages. The PuppetServer instance needs configuration
+for connecting to PuppetDB. Instructions on this configuration are
+below, but the official docs for this can be found
+[here](https://docs.puppet.com/puppetdb/4.3/connect_puppet_master.html).
+
+Update `~/.puppetlabs/etc/puppet/puppet.conf` to include:
+
+    [master]
+    storeconfigs = true
+    storeconfigs_backend = puppetdb
+    reports = store,puppetdb
```

Comment:
Should we bother suggesting to configure the `store` report terminus here or just use `reports = puppetdb`?